changelog.txt

```diff
@@ -15,6 +15,7 @@ Added
 +++++
 
 - Added the OCLF Crusher environment (#708).
+- Added feature to hide the status of the progress bar (#685).
```

Comment:
This needs to be moved to a new version entry, since 0.24.0 has already been released.